src/styles/main.scss

```diff
@@ -1,3 +1,215 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+$blue-color: #00acdc;
+$link-color: #060b35;
+
 body {
+  font-family: "Roboto", sans-serif;
+  box-sizing: border-box;
   margin: 0;
 }
+
+/* Header starts */
+
+.header {
```

Comment:
You should create for every block new sass file for better understanding your code. And also another sass file for variables. See more details in checklist, point 9.